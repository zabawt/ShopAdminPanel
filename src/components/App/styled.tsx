import styled from 'styled-components';

export const AppContainer = styled.div`
    max-width: 1366px;
    margin: auto;
    padding: ${props => props.theme.padding.medium} ${props=>props.theme.padding.large};
`;