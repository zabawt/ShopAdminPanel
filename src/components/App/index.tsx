import * as React from "react";
import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
import { ThemeProvider } from "styled-components";
import theme from "./../../commons/Theme";
import { AppContainer } from "./styled";
import AuthPage from "../AuthPage";

const App = (props: {}) => {
  return (
    <ThemeProvider theme={theme}>
      <AppContainer>
        <Router>
          <Link to="/">Admin panel</Link>
          <Switch>
            <Route path="/">
              <AuthPage />
            </Route>
          </Switch>
        </Router>
      </AppContainer>
    </ThemeProvider>
  );
};

export default App;
