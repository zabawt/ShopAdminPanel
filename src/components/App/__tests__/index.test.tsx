import * as React from "react";
import { shallow, mount } from "enzyme";
import App from "../index";

describe("<App/> tests", () => {
  it("renders without crashing", () => {
    shallow(<App />);
  });

  it("renders properly", () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
    expect(mount(<App />)).toMatchSnapshot();
  });
});
