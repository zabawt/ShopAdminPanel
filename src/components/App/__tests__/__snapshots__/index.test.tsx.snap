// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> tests renders properly 1`] = `ShallowWrapper {}`;

exports[`<App/> tests renders properly 2`] = `ReactWrapper {}`;
